next/src/components/footer/Footer.tsx

```diff
@@ -4,8 +4,27 @@ import Image from "next/image";
 import styled from "styled-components";
 
 import CSESocLogo from "../../../public/assets/logo_white.svg";
+import DiscordLogo from "../../../public/assets/socials/discord.svg";
+import FacebookLogo from "../../../public/assets/socials/facebook.svg";
+import InstagramLogo from "../../../public/assets/socials/instagram.svg";
+import YoutubeLogo from "../../../public/assets/socials/youtube.svg";
+import SpotifyLogo from "../../../public/assets/socials/spotify.svg";
```

Comment:
thoughts on making these absolute imports? since you guys have [`"baseUrl": "."` in your tsconfig](https://github.com/csesoc/website/blob/main/frontend/tsconfig.json#L22), you should be able to just import from `"public/assets/logo_white.svg"` without needing to traverse all the way back up the tree (this will also help if you ever want to move this component or copy these imports somewhere else)